Cython/Build/IpythonMagic.py

```diff
@@ -534,6 +556,15 @@ def clean_annotated_html(html):
         html = '\n'.join(l for l in html.splitlines() if not r.match(l))
         return html
 
+    @staticmethod
+    def code_to_html(code):
+        """Convert source code to a html-file, so tabs and new lines are properly shown
+        """
+        return '<!DOCTYPE html><html><body><span style="white-space: pre">\n' + \
+               code + \
```

Comment:
This should at least apply some escaping.

Reply:
True! Will take care of it, if it should be decided that this is the way to go.